src/main/java/by/exadel/internship/repository/FormRepository.java

```diff
@@ -38,4 +38,7 @@
     @Query(value = "UPDATE Form f SET f.deleted=true WHERE f.id= :formId")
     void deleteById(@Param("formId") UUID formId);
 
+    @Query(value = "select distinct f from Form f LEFT JOIN FETCH f.interview i where i.admin = :userId or i.techSpecialist = :userId")
+    List<Form> findDistinctByInterview_AdminAndInterview_TechSpecialist(@Param("userId")UUID userId);
```

Comment:
Why do you use so complicated name? I guess you tried doing query via to method name? :)
 Just
```suggestion
    List<Form> findAllByUserId(@Param("userId")UUID userId);
```

Reply:
yes i tried, thanks. done.